onnxmltools/convert/sklearn/operator_converters/FeatureUnion.py

```diff
@@ -0,0 +1,44 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ....proto import onnx_proto
+from ...common._apply_operation import apply_abs, apply_cast, apply_mul, apply_reshape, apply_sub
+from ...common._registration import get_converter, register_converter
+from .._parse import sklearn_operator_name_map
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.decomposition import TruncatedSVD
+
+class Output:
+    def __init__(self, full_name):
+        self.full_name = full_name
+
+class Oper:
+    def __init__(self, model, inputs, op_type, input_full_names, output_full_name):
+        self.raw_operator = model
+        self.inputs = inputs
+        self.input_full_names = input_full_names
+        self.type = op_type
+        self.outputs = [Output(output_full_name)]
+
+def convert_sklearn_feature_union(scope, operator, container):
+    op = operator.raw_operator
+    transform_result_name = []
+    supported_transformer_list = [PCA, TruncatedSVD]
+    for name, transform in op.transformer_list:
+        if type(transform) not in supported_transformer_list:
+            raise NotImplementedError
+        op_type = sklearn_operator_name_map[type(transform)]
+        result_name = scope.get_unique_variable_name('result')
```

Comment:
Looks like you didn't edit the parser but put all logics into `FeatureUnion`s converter. I perfer to create of ONNXMLTOOLS Operator during the parsing stage. This design somehow violates my original idea: parsing stage produces IR objects and those IR objects are converted at conversion stage. Let me try to add some details about a design in my mind.
1. Go to [this function](https://github.com/onnx/onnxmltools/blob/533c877194ed2c168f61219a03a4a094f5e1cd80/onnxmltools/convert/sklearn/_parse.py#L173) and add a new `_parse_sklearn_feature_union(scope, model, inputs)`
2. An implementation of `_parse_sklearn_feature_union(scope, model, inputs)`
```python
# scope is Scope in our IR
# model is FeatureUnion
# inputs is a string list with a single element
_parse_sklearn_feature_union(scope, model, inputs):
    # Output variable name of each transform. It's a list of string.
    transformed_result_names = []
    # Encode each transform as our IR objects
    for transform in model.transformer_list:
        transformed_result_names.add(_parse_sklearn_simple_model(scope, transform, inputs)
    # Create a Concat ONNX node
    concat_operator = scope.declare_local_operator('SklearnConcat')
    concat_operator.inputs = transformed_result_names

    # Declare output name of scikit-learn FeatureUnion
    union_name = scope.declare_local_variable('union')
    concat_operator.outputs.add(union_name)

    return union_name
```
3. Create a converter for `SklearnConcat` to `onnxmltools/convert/sklearn/operator_converters/SklearnConcat.`
4. Create a shape calculator for `SklearnConcat`.